samples/Commerce/Commerce.UI/App.xaml.host.cs

```diff
@@ -10,163 +10,154 @@ public sealed partial class App : Application
 	private static IHost BuildAppHost()
 	{
 		return UnoHost
-				.CreateDefaultBuilder()
+			.CreateDefaultBuilder()
 #if DEBUG
-				// Switch to Development environment when running in DEBUG
-				.UseEnvironment(Environments.Development)
+			// Switch to Development environment when running in DEBUG
+			.UseEnvironment(Environments.Development)
 #endif
```

Comment:
Why was this file touched - unless I missed something, there doesn't seem to be any changes, other than a bunch of formatting?